contact_desire_force: 2
demo_fx: -1
demo_fy: -1
xyz_vel_factor: [0.03, 0.03, 0.02]
rot_vel_factor: [0.3, 0.3, 0.3]
base_world_RPY: [0, 0, 0]
ef_wrench_factor: 0.2
ef_twist_factor: 0.04
ef_joint_factor_: 0.8
force_range0: 4
force_range1: 50
#1 changed to 0.3
torque_range: 0.3
period_time: 0.006
gravity_up_joints: [130, -90, -90, -90, -90, 90]
gravity_down_joints: [130, -90, -90, -90, 90, 90]
middle_jnt_: [130, -90, -90, -90, 0, 90]

pid_: [0.5, 3, 0]

hand_name_prefix: "robotiq"
UR_name_prefix: "UR5e"

name_suffix: 3517

